fabric/fabric.go

```diff
@@ -204,7 +204,15 @@ func (fps *FabricPaths) UnmarshalJSON(value []byte) error {
 								log.Fatalf("type assertion of staticVal %T headers '%v' to map[string]interafce{} was not ok", staticVal, staticVal)
 								continue
 							}
-							for k, v := range staticMap {
+
+							// sorted range through map to have stable results (test flapping)
+							keys := make([]string, 0, len(staticMap))
+							for k := range staticMap {
+								keys = append(keys, k)
+							}
+							sort.Strings(keys)
+							for _, k := range keys {
+								v := staticMap[k]
```

Comment:
If I get this code right we sort the keys of the `staticMap` and **reinsert** into `response.Headers` in order. However, if I get this right, Go does not have any order guarantees in terms of both map item insertion and iteration, so this operation seems to achieve nothing. Am I missing some other properties that make it work in this use case? Or maybe I misunderstood the purpose.